resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/rest/kubernetes/v2/ResourceStagingServiceRetrofit.scala

```diff
@@ -37,6 +37,10 @@ private[spark] trait ResourceStagingServiceRetrofit {
 
   @Streaming
   @retrofit2.http.GET("/api/v0/resources/{resourceId}")
```

Comment:
Based on our conversation offline, v1 will eventually go away and we'll just have this v2 submission. 
In a future PR, we will have YAML files to help quickly turn up a file submission server.